recipes/lighttpd/lighttpd-1.4.45/0001-Use-pkg-config-for-pcre-dependency-instead-of-config.patch

```diff
@@ -0,0 +1,37 @@
+From e7a8c925b9316a72bdc8f32789ffe56fda5c4788 Mon Sep 17 00:00:00 2001
+From: Alexander Kanavin <alex.kanavin@gmail.com>
+Date: Fri, 26 Aug 2016 18:20:32 +0300
+Subject: [PATCH] Use pkg-config for pcre dependency instead of -config script.
+
+RP 2014/5/22
+Upstream-Status: Pending
```

Comment:
I see, I didn't know they actually had defined meanings, and I've always read "Pending" more or less as meaning what "Submitted" means. I guess the latter is a better name anyway. Sorry for the noise, then.